vnext/Microsoft.ReactNative/XamlUIService.idl

```diff
@@ -24,6 +24,12 @@ namespace Microsoft.ReactNative
     DOC_STRING("Dispatches an event to a JS component.")
     void DispatchEvent(XAML_NAMESPACE.FrameworkElement view, String eventName, JSValueArgWriter eventDataArgWriter);
 
+    DOC_STRING(
+      "Invokes Yoga layout for a specific React tag. "
+      "This method is useful for views that use @IViewManagerRequiresNativeLayout "
+      "but have descendant views that require Yoga layout.")
+    void ApplyYogaLayout(Int64 tag, Single width, Single height);
```

Comment:
why is this in XamlUIService though? Seems more appropriate for the react context ?

Reply:
Fair enough. Wasn't sure where to put it.

Reply:
Perhaps a better approach would be to create a new IDL for UIManager module and expose that from ReactContext. We could then add things like GetReactRootView, ApplyYogaLayout, a callback for batch completion for native modules, etc.

Reply:
Another good question - why is `DispatchEvent` in XamlUIService? cc @acoates-ms. It looks to me like XamlUIService has become a dumping ground for APIs that we don't know where else to put :). I'll file a new issue to clean up XamlUIService (deprecate the API and move everything to APIs that are a bit more uniform in usage scenario).